src/road/traffic.py

```diff
@@ -66,6 +66,21 @@ def __init__(self, id, node: RoadSegmentNode):
         # Intersection
         self.wait_time = 0  # sec
 
+    @property
+    def speed(self):
+        """
+        Returns the speed of the vehicle in terms of tiles traveled
+        """
+        return self.vehicle_speed * tw
+    
+    @speed.setter
+    def speed(self, value):
+        """Sets the vehicle speed. Expects a value in terms of tile size"""
+        if isinstance(value, int):
+            self.vehicle_speed = value // tw # Floored for int
```

Comment:
I think what would be best is to accept the raw value and "clamp" it by new values in `settings.GameSettings`, like
`VEHICLE_SPEED_MIN` - minimum allowed vehicle speed
`VEHICLE_SPEED_MAX` - maximum allowed vehicle speed

`VEHICLE_SPEED_DEFAULT` could be added too and be used instead of the existing logic in `Vehicle.__init__()`.

I have tentative plans to make `settings.GameSettings` a singleton further down the line to allow the user to adjust the values at runtime and have the game logic automatically ingest the new values.